Update Rule6-3.md

xingcx opened the pull request.

JasonGlazer commented:
Original discussion at the link below. Please give this link time to load. This page might take a few seconds to load and another 4-8 seconds to land at the correct comment:

https://github.com/pnnl/ruleset-checking-tool/pull/19#discussion_r611820670

supriyagoel commented on docs/section9/Rule6-jl.md:
````diff
@@ -2,38 +2,51 @@
+      - Get the total design power_per_area for the space: ```space_lighting_power_per_area_proposed = sum( lighting.power_per_area for lighting in interior_lighting_proposed )```  
+
+      **Rule Assertion:** For each space that is Hotel/Motel Guestroom or Dormitory Living Quarters, or Dwelling Unites in the proposed model: ```space_lighting_power_per_area_proposed <= lighting_power_allowance_proposed```  
+
````
@xingcx and @JasonGlazer - should this be space_lighting_power_per_area_proposed >= lighting_power_allowance_proposed (greater than and not less than?)
App G says "lighting power used in the simulation shall be equal to the lighting power allowance in Table 9.6.1 for the appropriate space type or as designed, whichever is greater". so at the very minimum, the LPD would be equal to the the prescriptive requirement in Table 9.6.1 or greater than that value.

JasonGlazer replied:
I agree it should be >=

@xingcx please make that change.

mkarpman replied:
@JasonGlazer and @supriyagoel , shouldn't it be the maximum of the lighting power in U-RMR and lighting power allowance in Table 9.6.1 for that space type?

JasonGlazer replied:
@mkarpman that was our intention for the proposed change to use ">="

xingcx replied:
Updated the proposed value to the max of baseline allowance or user model lighting power.

supriyagoel commented on docs/section9/Rule6-jl.md:
````diff
@@ -2,38 +2,55 @@
+
+      - Get the total baseline power_per_area for the space: ```space_lighting_power_per_area_baseline = sum( lighting.power_per_area for lighting in interior_lighting_baseline )```  
+
+        **Rule Assertion:** For each space that is Hotel/Motel Guestroom or Dormitory Living Quarters, or Dwelling Units in the baseline model: ```space_lighting_power_per_area_baseline == lighting_power_allowance_baseline```  
````
@xingcx this looks perfect now! My only comment would be to split this into 2 RDS- one for the proposed and one for the baseline. This is because this rule has two rule assertions and we would like to have only one rule assertion for each RDS. You can just call the second one 6-jl-2. we will renumber all RDSs eventually.

mkarpman replied:
I agree

supriyagoel replied:
@mkarpman I agree. There will also be a separate RDS to check for controls in baseline and another one for proposed. (see the lighting TCDs).
@xingcx - the only change this one needs is to be sl=plit into 2 RDS. One for proposed for these specific space types and one for baselines for all space types.

xingcx commented:
I updated the rule based on the comments above. @supriyagoel @mkarpman @JasonGlazer 
- Changed rule number from 6-jl to 6-3
- Removed baseline related lines. I plan to cover baseline under Rule 6-8.

supriyagoel reviewed (approved):
this looks good.
